### ISSUE TITLE
TracePoint.new(:end) is not supported

### ISSUE THREAD

**Issue by segiddins** [oracle/truffleruby#3696]
It is called on cruby when a class or module block is closed

**Comment by eregon** [comment]
Do you need it for a particular gem or usage?

**Comment by segiddins** [comment]
Yes, joeldrapper/literal uses it to "freeze" classes once they're done being defined

**Comment by eregon** [comment]
Found it: https://github.com/joeldrapper/literal/blob/7f47ca228fd444d51696d4b4198e803c2069c1cb/lib/literal/enum.rb#L174-L180
Some thoughts on it: https://github.com/joeldrapper/literal/pull/93/files#r1814720822

**Comment by joeldrapper** [comment]
@eregon thank you. I replied to your comments in the thread on the literal repo. I think we can just disable this feature for TruffleRuby. It’s not essential, it’s just nice to know that Enums are constant — defined once at boot and never updated during runtime.

**Comment by eregon** [comment]
> I think we can just disable this feature for TruffleRuby.

That would be nice.
A good way to detect that is:
```ruby
has_end_tracepoint = begin
  TracePoint.new(:end) {}
  true
rescue ArgumentError
  false
end
```

In terms of priority, implementing `TracePoint.new(:end) {}` is fairly low at this point, because it seems rarely used, non-trivial effort and there are simply more important compatibility issues.
